docs/adrs/0076-use-playwright.md

```diff
@@ -0,0 +1,85 @@
+
+Unfortunately, several of our existing integration tests experience sporadic failures when run by CI. At the current time of writing (January 2023), 3-5% of CI builds experience a failure that resolves itself if immediately re-run. Several efforts have been undertaken to resolve, or at least reduce, the level of test "flakiness" over the past two years, which have only seen temporary success. The limiting reagent has been primarily our lack of in-house experience with implementing Cypress best practices and resolving race conditions, given a low level of information provided by the output of the (sporadically) failing Cypress tests.
+
+Like other third-party code libraries, Cypress receives periodic version updates from its maintainers in order to improve performance and/or implement bug fixes; these updates may include security fixes. For the past year, our project has been stuck with using version 8.5.0 (released in [September 2021](https://docs.cypress.io/guides/references/changelog#8-5-0)), as using any further versions in our project caused an elevated rate of test flakiness, to the point where it would functionally hinder or block development.
+
+In July 2022, an effort was undertaken to [upgrade the version of Cypress to 10.3.1](https://github.com/transcom/mymove/pull/8934), which was the current version at the time. It was discovered that with the upgrade to version 10, a non-trivial amount of file renaming and restructuring work would need to be performed; once that was completed, [the rate of test failures increased significantly](https://docs.google.com/spreadsheets/d/1AsBB8_EkQ1ltqM2a4zfdDs4EvTfqdN76slUrdXeH5LA/edit#gid=0). Several existing and currently passing tests were now failing on each execution, with others now failing at an incidence rate of 10-20% when run as a part of our CI process. This was investigated, but ultimately set aside in the interest of feature development work.
+
+In December 2022, a renewed effort was undertaken to [upgrade the version of Cypress to 12.0.2](https://github.com/transcom/mymove/pull/9737), which was now the current version. The same file renaming and restructuring work needed to be performed; once finished, [the rate of test failures skyrocketed](https://docs.google.com/spreadsheets/d/1P2qvEZqdW9Wyg5McYAe7GNt64abSKjSU3pFhIsJneZE/edit#gid=1060625568). The number of both consistently failing and sporadically failing tests had approximately doubled from the effort six months earlier, and now affected at least half of all integration tests in our existing suite.
```

Comment:
**question**: I don't have access to the first google sheets link (https://docs.google.com/spreadsheets/d/1AsBB8_EkQ1ltqM2a4zfdDs4EvTfqdN76slUrdXeH5LA/edit#gid=0) and the second one seems to be open to anyone with the link (https://docs.google.com/spreadsheets/d/1P2qvEZqdW9Wyg5McYAe7GNt64abSKjSU3pFhIsJneZE/edit#gid=1060625568). Should they both just be open to anyone with the link? Both the mymove repo and this docs repo are open source so that feels like it's probably fine, but I figured I'd ask in case anyone thinks it should be different.